Rasa_Bot/data/rules/rules_notifications.yml

```
@@ -0,0 +1,56 @@
+version: "3.1"
+
+rules:
+## Tracking notification
+- rule: send tracking notification
+  steps:
+  - intent: EXTERNAL_trigger_track_notification  # TODO: set trigger
+  - action: utter_notifications_tracking_1
+  - action: utter_notifications_tracking_2
+  - action: utter_notifications_tracking_3
+
+## Step goal notification
+- rule: send step goal notification
+  steps:
+  - intent: EXTERNAL_pa_step_goal_notification  # TODO: set trigger
+  - action: action_notifications_get_daily_step_goal_from_db
+  - action: utter_notifications_step_goal_1
+  - action: utter_notifications_step_goal_2
+
+## finish dialog notification
+- rule: finish dialog notification
+  steps:
+  - intent: EXTERNAL_finish_dialog_notification  # TODO: set trigger if user is inactive for 1hour
+  - action: utter_notifications_finish_dialog_1
+  - action: utter_notifications_finish_dialog_2
+
+## Inactive users notification
+- rule: inactive users notification
+  steps:
+  - intent: EXTERNAL_inactive_user_notification  # TODO: set trigger if user inactive for 10 days
+  - action: utter_notifications_user_inactive_1  # TODO: go to utter_central_mode_options
+  - action: utter_central_mode_options
```

```suggestion
  - action: utter_notifications_user_inactive_1
  - action: action_select_menu
```

We can use `action_select_menu`. It determines whether a dialog has to be completed, and shows the right menu